python/ray/experimental/features.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray
+
+
+def flush_redis_unsafe():
```

Comment:
Add some documentation on what this is doing, in what sense it is unsafe and when it can safely be used. If we only want to flush logs there even in the future, we should rename it to "flush_redis_logs".